engine/src/main/java/com/redhat/cloud/notifications/utils/CloudEventParser.java

```diff
@@ -0,0 +1,126 @@
+package com.redhat.cloud.notifications.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.ApplyDefaultsStrategy;
+import com.networknt.schema.JsonMetaSchema;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaException;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SchemaValidatorsConfig;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationResult;
+import com.networknt.schema.ValidatorTypeCode;
+import com.networknt.schema.uri.URIFactory;
+import com.networknt.schema.uri.URLFactory;
+import com.redhat.cloud.event.core.v1.RHELSystem;
+import com.redhat.cloud.notifications.models.ConsoleCloudEvent;
+import com.redhat.cloud.notifications.validator.LocalDateTimeValidator;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+
+@ApplicationScoped
+public class CloudEventParser {
```

Comment:
Could this class be moved to [event-schemas-java](https://github.com/RedHatInsights/event-schemas-java)?

Reply:
I guess we could do that.